app/PharBuilder.php

```diff
@@ -455,7 +455,7 @@ protected function addDir($directory, $excludes = array())
              *
              * @var \Symfony\Component\Finder\SplFileInfo $file
              */
-            $this->addFile($directory . DIRECTORY_SEPARATOR . $file->getRelativePathname());
+            $this->addFile(rtrim($directory, '\/') . DIRECTORY_SEPARATOR . $file->getRelativePathname());
```

Comment:
I don't know, but it fails on windows. Be able to test again later, so If you need can give information.